index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <header class="flex-container">
+      <div class="logo-container">
+        <a class="logo" href="">My Logo</a>
+      </div>
+      <nav class="nav_item">
+        <ion-icon class="menu" name="menu-sharp"></ion-icon>
+      </nav>
+    </header>
+
+    <section class="hero">
+      <h2 class="name">
+        I'm Nelofar Zabi<br />Glad to see<br />
+        you!
+      </h2>
```

Comment:
Weldone on implementing some of the project requirements, please the main title of your webpage should be a `h1` property for semantics reasons and search engine optimization 😄